--- origsrc/kdb-3.0.1/src/tools/KDbUtils_p.h	2017-03-09 03:20:11.234698300 -0600
+++ src/kdb-3.0.1/src/tools/KDbUtils_p.h	2017-07-18 02:53:21.869894000 -0500
@@ -44,7 +44,7 @@ void qAsConst(const T &&) Q_DECL_EQ_DELE
 #endif // QT_VERSION < 0x050700
 
 //! @def KDB_SHARED_LIB_EXTENSION operating system-dependent extension for shared library files
-#if defined(Q_OS_WIN)
+#if defined(Q_OS_WIN) || defined(Q_OS_CYGWIN)
 #define KDB_SHARED_LIB_EXTENSION ".dll"
 #elif defined(Q_OS_MAC)
 // shared libraries indeed have a dylib extension on OS X, but most apps use .so for plugins
